react/features/base/ui/components/web/ContextMenu.tsx

```diff
@@ -100,7 +100,7 @@ interface IProps {
     onMouseLeave?: (e?: React.MouseEvent) => void;
 }
 
-const MAX_HEIGHT = 400;
+const MAX_HEIGHT = 500;
```

Comment:
Why do we need to change this height? I think we recently made adjustments to better accommodate small screens.

Reply:
It remained like this when testing the feature. With 400 I had to scroll for the custom button added, so I set it up to 500

Reply:
I'll just set it back to 400